### PR TITLE
[9.x] Add validatedExcept method to FormRequest

### PR DESCRIPTION
**Pull request by AbbasShDev**
This PR adds validatedExcept method to FormRequest to get the validated data from the request except for a specified array of items.

### CODE REVIEW

File: src/Illuminate/Foundation/Http/FormRequest.php [hunk 1]
```diff
@@ -215,6 +216,23 @@ public function validated($key = null, $default = null)
         return data_get($this->validator->validated(), $key, $default);
     }
 
+    /**
+     * Get the validated data from the request except for a specified array of items.
+     *
+     * @param  array|mixed  $keys
```
Comment (driesvints) [inline src/Illuminate/Foundation/Http/FormRequest.php:222]: `mixed` can never be combined with something else.

Comment (AbbasShDev) [reply src/Illuminate/Foundation/Http/FormRequest.php]: Thank you @driesvints, fixed

**Comment by taylorotwell** [comment]
Thanks for your pull request to Laravel!

Unfortunately, I'm going to delay merging this code for now. To preserve our ability to adequately maintain the framework, we need to be very careful regarding the amount of code we include. 

If possible, please consider releasing your code as a package so that the community can still take advantage of your contributions! 

If you feel absolutely certain that this code corrects a bug in the framework, please "**@**" mention me in a follow-up comment with further explanation so that GitHub will send me a notification of your response.

**Comment by igorgaming** [comment]
@AbbasShDev you can `$request->safe()->except(...)`

**Comment by AbbasShDev** [comment]
Thnaks @igorgaming, that is excatly what I want

**Pull request closed without merging**